### DISCUSSION TITLE
context.duration extension has been removed...

### DISCUSSION THREAD

**Discussion by saitkuzu** [VB10/kartal#69] [General]
Hi everybody. I use kartal as an essential on my project. It is a powerful and developing package. However, there is a big problem. Its structure has changed many times without any information. The last change is **duration**. I agree there is a new dart method, but it's not good removing a method suddenly. Somebody should note these changes and publish them in a version change log.

I hope you consider my suggestion.

**Comment by VB10** [comment]
thank you for sharing, @saitkuzu. I have removed the context.duration property as you mentioned. Thanks!
